### PR TITLE
Add: a rudimentary README

### PR DESCRIPTION
**Pull request by larswirzenius**
You didn't indicate a license so I left that empty.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,33 @@
+fling -- transfer data from stdin over network to destination quickly
+-----------------------------------------------------------------------------
+
+fling transfers data quickly over a trusted network. It does not
+encrypt the data.
+
+You need to run fling on both ends of the transfer. Run it first on
+the receiver:
+
+    fling 12756 > file.dat
```
Comment (rjek) [inline README:10]: I think this is missing a -r

File: README [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+And then on the sender:
+
+    flint other.host.address 12756 < file.dat
```
Comment (rjek) [inline README:14]: s/flint/fling/

File: README [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Linux by running the `make` command. Copy the `fling` binary to the
+target host, or build a fresh copy there.
+
+
+Legalese
+-----------------------------------------------------------------------------
+
+Copyright 2019  Rob Kendrick
+
+License has not been specified. FIXME.
```
Comment (rjek) [inline README:33]: I've pushed a change to the sources, including copyright statement and licence.

**Comment by larswirzenius** [comment]
I fumble and will open a new one.

**Pull request closed without merging**